### ISSUE TITLE
 java.lang.IllegalArgumentException in com.cardinalcommerce.cardinalmobilesdk.Cardinal.cca_continue

### ISSUE THREAD

**Issue by maksimdyakonov** [braintree/braintree_android#724]
### Braintree SDK Version

4.27.1

### Environment

Production

### Android Version & Device

Xiaomi Redmi Note 10 PRO (M2101K6G) - Android 11

### Braintree dependencies

implementation 'com.braintreepayments.api:card:4.27.1'
implementation 'com.braintreepayments.api:data-collector:4.27.1'
implementation 'com.braintreepayments.api:three-d-secure:4.27.1'

### Describe the bug

The app crashes during the 3D Secure payment with java.lang.IllegalArgumentException. Payment is not executed

### To reproduce

Normal 3DS payment flow
**Stacktrace**
```
Exception java.lang.RuntimeException:
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3521)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3693)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:85)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2135)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:236)
  at android.app.ActivityThread.main (ActivityThread.java:8059)
  at java.lang.reflect.Method.invoke
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:656)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:967)
Caused by java.lang.IllegalArgumentException: bad base-64
  at android.util.Base64.decode (Base64.java:163)
  at android.util.Base64.decode (Base64.java:138)
  at android.util.Base64.decode (Base64.java:120)
  at com.cardinalcommerce.cardinalmobilesdk.Cardinal.cca_continue (Cardinal.java)
  at com.braintreepayments.api.CardinalClient.continueLookup (CardinalClient.java)
  at com.braintreepayments.api.ThreeDSecureActivity.onCreateInternal (ThreeDSecureActivity.java)
  at com.braintreepayments.api.ThreeDSecureActivity.onCreate (ThreeDSecureActivity.java)
  at android.app.Activity.performCreate (Activity.java:8157)
  at android.app.Activity.performCreate (Activity.java:8129)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1310)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3494)

### Expected behavior

 Payment is executed

### Screenshots

_No response_

**Comment by sshropshire** [comment]
Hi @maksimdyakonov thanks for using the Braintree SDK for Android. We have a[ fix for this](https://github.com/braintree/braintree_android/blob/master/CHANGELOG.md#unreleased) that will be deployed soon.

**Comment by hollabaq86** [comment]
We just released 4.27.2 which should resolve this issue!

**Comment by sshropshire** [comment]
@maksimdyakonov as @hollabaq86 stated please update to version [4.27.2](https://github.com/braintree/braintree_android/blob/master/CHANGELOG.md#4272) to obtain the fix.
